rust/agama-lib/src/bootloader/model/bootloader.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) [2024] SUSE LLC
```

Comment:
Why `bootloader::model::bootloader`? If the model is simple enough, I would go for just `bootloader::model`.

Reply:
agreed. Same explanation as above.